runbooks/addonfactory-workflow-addon-release-docker-images.md

```diff
@@ -0,0 +1,27 @@
+# Runbook to creating and publishing docker images used in reusable workflow
+## Runbook to publish multiple images of different Linux flavors and versions for scripted inputs tests
+Once there is new Splunk release, and [matrix](https://github.com/splunk/addonfactory-test-matrix-action) is updated, we need to make sure that Splunk images for scripted inputs tests are created and published.
+### Steps
+
+#### Update OS images
+- check what OSs are listed in definition of matrix for scripted inputs tests [here](https://github.com/splunk/addonfactory-workflow-addon-release/blob/72497e5c03894369b8fbdd2a2c4134c233ba1b5d/.github/workflows/reusable-build-test-release.yml#L27)
+- if any is missing in [ta-automation-docker-images](https://cd.splunkdev.com/taautomation/ta-automation-docker-images/-/tree/main/dockerfiles) then add new Dockerfile
+
+#### Create images and publish them to ECR
+- figure out what version of Splunk is needed (sha) using go/fetcher
+- trigger [pipeline](https://cd.splunkdev.com/taautomation/ta-automation-docker-images/-/pipelines/new) for every OS flavor separately
+
+## Runbook to publish unreleased splunk image for testing
```

Comment:
`Splunk` instead of `splunk`? Same in the following lines?

Reply:
Done.